nss-day-cohort-29/nutshell-spirit-wind
# Db register

# Description

I've added a registration form and button


## Type of change

Please delete options that are not relevant.

- [ X ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

git fetch --all
git checkout db-register
cd into src/lib
run grunt
enter text into register fields
click Register User Button
input will appear in console

# Checklist:
